src/TestSuite/ConsoleIntegrationTestTrait.php

```diff
@@ -88,6 +89,10 @@ public function exec($command, array $input = [])
 
         try {
             $this->_exitCode = $runner->run($args, $io);
+        } catch (MissingConsoleInputException $e) {
+            $messages = $this->_out->messages();
+            $e->setQuestion($messages[count($messages) - 1]);
```

Comment:
I guess this could fail with 0 messages, but if it isn't an issue in test cases then no worries.

Reply:
I can guard against that scenario. It could happen if someone tried hard.